projects/ama-sdk/src/lib/helpers/common.ts

```diff
@@ -39,3 +39,12 @@ export interface EntityDialogPayload {
     values?: EntityDialogForm;
     action: any;
 }
+
+export const sanitizeString = (text: string) => {
```

Comment:
1.) Please create a utils directory under the common folder, where we can store utility functions like this, per file. If I can ask, as a refactoring please put these files into this directory:
- blob2json.ts
- create-entries-names.ts with its spec
- empty-diagram.ts with its spec

And from this point on, we can organise utility helper function into this directory.
2. ) Where are the unit tests?? We need to write unit tests, no matter what, especially for this kind of easy to test functionalities. We are supposed to follow TDD, basically we should not write any code before writing tests for it.